packages/react/src/components/Lightbox/README.md

```diff
@@ -0,0 +1,104 @@
+# Lightbox
+
+> The Lightbox component is to be utilized within IBM.com.
+
+## Getting started
+
+Here's a quick example to get you started.
+
+```scss
+// yourapplication.scss
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+```
+
+> 💡 Only import font's once per usage
+
+```javascript
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { LightBox } from '@carbon/ibmdotcom-react';
+import '@carbon/ibmdotcom-styles/scss/components/lightbox/_lightbox.scss';
```

Comment:
This needs to be moved up to the `scss` block

Reply:
updated with [this commit](https://github.com/carbon-design-system/ibm-dotcom-library/pull/882/commits/afbd414592d1948f0861ff4866931e632f6389c6)